src/napari_topostats/_tests/test_widget.py

```diff
@@ -1,66 +1,110 @@
-import numpy as np
-
-from napari_topostats._widget import (
-    ExampleQWidget,
-    ImageThreshold,
-    threshold_autogenerate_widget,
-    threshold_magic_widget,
+"""Tests for the widget functionalities of the plugin."""
+
+# pylint: disable=redefined-outer-name
+from pathlib import Path
+
+import pytest
+from napari_afmreader._reader import reader_function
+from pytestqt.qtbot import QtBot
+from qtpy.QtCore import Qt
+
+from napari_topostats._widget import AVAILABLE_FUNCTIONS
+
+# --- Helper Functions ---
+
+
+def load_test_image(viewer, image_path):
```

Comment:
I wonder if these helper functions defined both here and in `test_io.py` might be required across the test-suite? If so would it be possible to have them as fixtures or inherited in some manner?